plugins/src/test/kotlin/org/gradle/kotlin/dsl/plugins/embedded/EmbeddedKotlinPluginTest.kt

```diff
@@ -116,20 +114,38 @@ class EmbeddedKotlinPluginTest : AbstractPluginTest() {
                 ${dependencyDeclarationsFor("compile", listOf("stdlib", "reflect"))}
             }
 
-            configurations["compileClasspath"].files.map { println(it) }
-
-            tasks {
-                "downloadAuxiliaries" {
-                    val depsExtractor = IdeDependenciesExtractor()
-                    val deps = depsExtractor.extractRepoFileDependencies(dependencies,
-                                                                         listOf(configurations["compileClasspath"]),
-                                                                         emptyList(),
-                                                                         true, true)
-                    deps.forEach { it.sourceFiles.forEach { println(it.name) } }
-                    deps.forEach { it.javadocFiles.forEach { println(it.name) } }
-                }
-            }
-
+            configurations["compileClasspath"].files.forEach {
+                println(it)
```

Comment:
Do you want to leave this here?

Reply:
This was in [v0.14.2](https://github.com/gradle/kotlin-dsl/tree/v0.14.2) and has been removed on `develop` since then. Given it's a test I would leave it there yes.